test/Datadog.Trace.ClrProfiler.Managed.Tests/IntegrationSignatureTests.cs

```diff
@@ -2,12 +2,16 @@
 using System.Linq;
 using System.Reflection;
 using Datadog.Trace.ClrProfiler.Integrations;
+#if !NETCOREAPP2_1 && !NETCOREAPP3_0 && !NETCOREAPP3_1
```

Comment:
```suggestion
#if !NETCOREAPP
```

Reply:
Here's a reference to the preprocessor directives which can make the expression of `NETCOREAPP` a more concise evaluation: https://docs.microsoft.com/en-us/dotnet/core/tutorials/libraries

Reply:
Much better, thanks!

Reply:
Nice!